### PR TITLE
Keyboard Shortcut

### LINKED ISSUE #8: Add a keyboard shortcut to convert the currency
We should have a keyboard combo to convert the article currency to our intended currency value without actually clicking the extension icon, which may be hidden from the extension menu.

### PR DESCRIPTION
**Pull request by therealshabi**
Closes #8 
## Changelogs:-
- Add a keyboard shortcut to run the currency convertor
- Fix linting errors/typos.
- Update Readme with some useful information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # Realtime-Currency-Convertor
-Browser Extension to convert foreign currency to local currency, realtime.
+
+Browser Extension to convert foreign currency to local currency, real time.
+
+## How to use:-
+
+- Navigate to any article which has a different currency mentioned in its article body, for example in [this](https://www.smartprix.com/bytes/samsung-galaxy-note-10-realme-5-pro-realme-5-mi-a3-launch-in-india-next-week-heres-what-to-expect/) article the prices are mentioned in dollars and we want them in say Rupees.
```
Comment (rajatgoyal715) [inline README.md]: @therealshabi Are you sure you want to give Smartprix link in the example?

**Comment by rajatgoyal715** [comment]
@therealshabi How are we planning to manage this and #6 together?

File: README.md (hunk 1 above)
Comment (therealshabi) [reply README.md]: Yes so what that's only an article in which our extension can be used

**Comment by therealshabi** [comment]
> @therealshabi How are we planning to manage this and #6 together?

If we are using more than one target currency, we can set one as `default` and the shortcut would convert the string to that default target currency option. Obviously we can't show multiple currencies anyways (not our intention)

File: README.md (hunk 1 above)
Comment (therealshabi) [reply README.md]: If you have any other test article, you may suggest that as well, I don't think so there is a need though

File: src/js/background.js [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+chrome.commands.onCommand.addListener(command => {
+	if (command === "convert_currency") {
+		chrome.tabs.query({
+			active: true,
+			currentWindow: true
+		}, tabs => executeScript(tabs[0].id));
+	}
+});
+
+function executeScript(id) {
```
Comment (ajeet97) [inline src/js/background.js:1]: Can move this function to the top.

File: src/js/background.js [hunk 3]
```diff
@@ -1,7 +1,18 @@
-chrome.browserAction.onClicked.addListener((tab) => {
-	chrome.tabs.executeScript(tab.id, {
+chrome.browserAction.onClicked.addListener(tab => executeScript(tab.id));
+
+chrome.commands.onCommand.addListener(command => {
+	if (command === "convert_currency") {
```
Comment (ajeet97) [inline src/js/background.js]: early return, if command is not `convert_currency`

File: README.md (hunk 1 above)
Comment (ajeet97) [reply README.md]: Shouldn't it be
> Navigate to any `page` ??

**Comment by ajeet97** [comment]
@therealshabi, can you add installation part in README as mentioned in #2.

File: README.md (hunk 1 above)
Comment (therealshabi) [reply README.md]: okay 👍

File: src/js/background.js (hunk 2 above)
Comment (therealshabi) [reply src/js/background.js:1]: Hoisting?

**Comment by therealshabi** [comment]
> @therealshabi, can you add installation part in README as mentioned in #2.

Ain't the installation part obvious? We don't have to do anything outrageous for that, simple extension installing? Moreover, I guess the Readme part is out of scope for this PR, we can make a separate PR for the same, solving #2 altogether. I added the current part in Readme so that we don't forget how the Keyboard shortcut will actually work

File: src/js/contentScript.js [hunk 4]
```diff
@@ -15,8 +15,7 @@
 
 			if (top.nodeType === Node.ELEMENT_NODE && regex.test(top.innerHTML)) {
 				top.innerHTML = top.innerHTML.replace(regex, replaceMethod.bind(top));
-			}
-			else if (top.nodeType === Node.TEXT_NODE && regex.test(top.nodeValue)) {
+			} else if (top.nodeType === Node.TEXT_NODE && regex.test(top.nodeValue)) {
```
Comment (rishabhsingh971) [inline src/js/contentScript.js:18]: I think we should make an eslint config for this project.. so that we all are on the same page.

**Review by rishabhsingh971 — APPROVED** [review]

**Review by ajeet97 — APPROVED** [review]

File: src/js/background.js (hunk 2 above)
Comment (ajeet97) [reply src/js/background.js:1]: Just to follow conventions. (Declaration is before usage)

**Pull request merged**